exporter/fileexporter/exporter_test.go

```diff
@@ -104,6 +141,48 @@ func TestFile_Export(t *testing.T) {
 				content:       "./testdata/custom_csv_format.csv",
 			},
 		},
+		{
+			name:    "complex parquet value",
+			wantErr: false,
+			fields: fields{
+				Format:                  "parquet",
+				ParquetCompressionCodec: parquet.CompressionCodec_SNAPPY.String(),
+			},
+			args: args{
+				featureEvents: []exporter.FeatureEvent{
+					{
+						Kind:         "feature",
+						ContextKind:  "anonymousUser",
+						UserKey:      "ABCD",
+						CreationDate: 1617970547,
+						Key:          "random-key",
+						Variation:    "Default",
+						Value: map[string]interface{}{
+							"string": "string",
+							"bool":   true,
+							"float":  1.23,
+							"int":    1,
+						},
+						Default: false,
+					},
+				},
+			},
+			expected: expected{
+				fileNameRegex: "^flag-variation-" + hostname + "-[0-9]*\\.parquet$",
+				featureEvents: []exporter.FeatureEvent{
+					{
+						Kind:         "feature",
+						ContextKind:  "anonymousUser",
+						UserKey:      "ABCD",
+						CreationDate: 1617970547,
+						Key:          "random-key",
+						Variation:    "Default",
+						Value:        "map[bool:true float:1.23 int:1 string:string]",
```

Comment:
in parquet the interface Value is not exactly similar like the input, it got stringify